sajibportfolio.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="/sajibportfolio.css" />
```

Comment:
please think you should remove the forward slash symbol in `href="/sajibportfolio.css" />` e.g `href="sajibportfolio.css" />` because it is preventing styles from applying to the HTML layout.

Reply:
done

Reply:
Fixed.